wordpress-mobile/WordPress-Android
# Feature/15532 about screen animation on logos

Fixes #15532

This PR adds physic animation on Unified About Screen for Automattic family of logos using [Jbox2D](https://github.com/jbox2d/jbox2d) a  physics engine with very small footprint (~ 230 KB).  More on this [here](https://proandroiddev.com/how-to-create-a-bubble-selection-animation-on-android-627044da4854)

https://user-images.githubusercontent.com/990349/140690579-74d8a35d-14e9-469a-9e8d-1a5eaf0b23b8.mp4


To test:

- Navigate to Me -> App Settings -> Scroll down and click on WordPress for Android
- See Automattic family apps logos appear animated naturally bouncing and floating about
- Click on a logo, it should launch the respective app web page

Please ignore the logo shapes, will be updated to right size and shape

NOTE:  This PR is raised as draft, will be integrated into a separate lib in this [repo](https://github.com/Automattic/about-automattic-android).  

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
